zerver/lib/mention.py

```diff
@@ -43,6 +43,12 @@ def Q(self) -> Q:
             raise AssertionError("totally empty filter makes no sense")
 
 
+@dataclass
+class MentionText:
+    text: Optional[str]
+    is_wildcard: bool
+
```

Comment:
Would `TypedDict` be a simpler way to do this (especially since we don't intend to add any methods to this dataclass)?

Reply:
I think a dataclass is fine fo rhtis.